File: Domain/Kontrolla.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Domain
{
    public class Kontrolla
    {
        public int Id { get; set; }
        public string EmriKontrolles { get; set; }
        public float Kosto { get; set ; }
        public int TerminiId { get; set; }
        public Termini Termini { get; set; }
        public ICollection <Tretmani> Tretmanet { get; set; } = new List<Tretmani>();
    }
}

File: client-app/src/app/models/kontrolla.ts
import { Pacienti } from "./pacienti";
import { Termini } from "./termini";
import { Tretmani } from "./tretmani";

export interface Kontrolla {
    id?: string;
    emriKontrolles: string;
    kosto: string;
    pacientiId?: string;
    pacienti?: Pacienti;
    tretmaniId?: string;
    tretmani?: Tretmani;
    terminiId: string;
    termini?: Termini;
  }